### DISCUSSION TITLE
Getting GitHub ratelimit error on last step of install (adding custom repo in HACS)

### DISCUSSION THREAD

**Discussion by sunebuhl** [fuatakgun/eufy_security#137] [Q&A] [answered]
Hi all. 

I'm trying to get this wonderful eufy_security integration up and running on my HomeAssistant. 

Following the guide have been without issues, howeever I hit a "GitHub ratelimit error" when trying to add the eufy security custom repo in step 6.3 (Installing Integration). 

Log file shows: <Integration fuatakgun/eufy_security> GitHub Ratelimit error
17:57:42 – (ERROR) HACS (custom integration) - message first occurred at 17:47:39 and shows up 4 times

With log details: 

Logger: custom_components.hacs
Source: custom_components/hacs/helpers/functions/validate_repository.py:67
Integration: HACS (documentation, issues)
First occurred: 17:47:39 (4 occurrences)
Last logged: 17:57:42

<Integration fuatakgun/eufy_security> GitHub Ratelimit error

I'm new to HACS (and even GitHub + Home Assistant) so hoping someone can point me to where the error is. :-)

Thanks,

**Comment by fuatakgun** [comment, marked as answer]
When you first install HACS first time, it loads many repositories in background, so you hit the limit and it is fine. You just need to be patient, but nothing about this integration
